tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -5908,7 +5925,11 @@ TEST_F(CompilerTest, QuickLlTest) {
 	CodeGenTestCheckBatchDir(L"..\\CodeGenHLSL\\quick-ll-test");
 }
 
+#ifdef _WIN32
 TEST_F(CompilerTest, SingleFileCheckTest) {
+#else
+TEST_F(CompilerTest, DISABLED_SingleFileCheckTest) {
```

Comment:
It's basically disabled on windows too, TAEF just has a different mechanism to do it. It can be run if you pass the right flag and provide the necessary additional parameter. The same is true of TAEF, it's just a different flag.

Reply:
Also of note, this isn't run as part of the standard testing on Windows. As far as I know, it never is except maybe for local testing of a specific issue. I think it's just a backdoor to plug in a file with the comment commands and correctness checks without having to add it to any specific tests.